Vagrantfile

```diff
@@ -118,7 +132,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   end
 
   if config.vm.networks.any? { |type, options| type == :private_network }
-    config.vm.synced_folder ".", "/vagrant", type: "nfs", mount_options: ['rw', 'vers=3', 'tcp', 'actimeo=1']
```

Comment:
These attributes were causing the filesystem to hang on my Mac. (2018 Macbook Pro; macOS Ventura 13.0.1; Vagrant 2.3.3; VirtualBox 7.0.4)

vers=3 should be the default and redundant anyway

Reply:
removing `, 'vers=3', 'tcp'` broke the folder mapping on my linux box, I had to add it back

first I was getting "requested NFS version or transport protocol is not supported"

after restarting the NFS service that error went away, the `/vagrant` folder started showing up inside the VM, but it was empty

only after adding `, 'vers=3', 'tcp'` back was `/vagrant` properly mapped

Reply:
Hmm. Well, the same crashes it on my Mac... Do you have to add both attributes in order for it to work? From what I understand, `vers=3` should be a default option. I'm not sure about `tcp`.

Reply:
adding back only `tcp` seems to make the difference, I still had to restart the NFS service (maybe that's an unrelated issue), but once restarted the `/vagrant` folder was mapped as expected